joint1:
  ID: 1
  Baud_Rate: 4000000
  Return_Delay_Time: 0
  Operating_Mode: 3
  Profile_Acceleration: 1606
  Profile_Velocity: 1460
  Position_P_Gain: 40
  Position_I_Gain: 0
  Position_D_Gain: 0
joint2:
  ID: 2
  Baud_Rate: 4000000
  Return_Delay_Time: 0
  Operating_Mode: 3
  Profile_Acceleration: 1606
  Profile_Velocity: 1460
  Position_P_Gain: 20
  Position_I_Gain: 0
  Position_D_Gain: 0
joint3:
  ID: 3  
  Baud_Rate: 4000000
  Return_Delay_Time: 0
  Operating_Mode: 3
  Profile_Acceleration: 1606
  Profile_Velocity: 1460
  Position_P_Gain: 20
  Position_I_Gain: 0
  Position_D_Gain: 0
joint4:
  ID: 4
  Baud_Rate: 4000000
  Return_Delay_Time: 0
  Operating_Mode: 3
  Profile_Acceleration: 90
  Profile_Velocity: 50
  Position_P_Gain: 150
  Position_I_Gain: 0
  Position_D_Gain: 0
joint5:
  ID: 5
  Baud_Rate: 4000000
  Return_Delay_Time: 0
  Operating_Mode: 3
  Profile_Acceleration: 90
  Profile_Velocity: 75
  Position_P_Gain: 150
  Position_I_Gain: 0
  Position_D_Gain: 0
joint6:
  ID: 6
  Baud_Rate: 4000000
  Return_Delay_Time: 0
  Operating_Mode: 3
  Profile_Acceleration: 125
  Profile_Velocity: 105
  Position_P_Gain: 200
  Position_I_Gain: 0
  Position_D_Gain: 0
